ui/src/components/common/AppItem.jsx

```diff
@@ -20,14 +20,29 @@ class AppItem extends PureComponent {
         }}
       />
     );
-    return (
+    const ftm_message = (
+      <FormattedMessage
+        id="footer.aleph"
```

Comment:
This is probably not right, but I wasn't sure if I can freestyle here. would `id="footer.ftm"` work?

Reply:
Was just going to comment about this! This is used as a key in translation files (e.g. [this one](https://github.com/alephdata/aleph/blob/main/ui/i18n/translations/raw/de.json)) (and also by Transifex). You can use any string you want as long as it’s unique, just try to stay consistent. `footer.ftm` should be good!